de.json

```
@@ -2188,29 +2188,29 @@
-  "card_subscription_fee_per_type": "Subscription: {{fee}} per {{type}}",
-  "reorder_vehicles": "Reorder vehicles",
-  "edit_order": "Edit order",
-  "amenity_form_timing": "Make a stop",
-  "delete_vehicle_reset_account": "Reset account",
-  "delete_vehicle_reset_account_description": "This will delete all your data (including vehicles) permanently. Are you sure?",
-  "delete_non_tlm_vehicles": "Delete non-TLM vehicles",
-  "send": "Send",
-  "navigation_go_home": "Go to home",
-  "departure_soc_description_detailed": "The starting battery level (State of Charge/SoC) of your car when beginning the journey.",
-  "destination_arrival_soc_description_detailed": "The minimum battery level (State of Charge/SoC) allowed when arriving at your final destination.",
-  "charger_arrival_soc_description_detailed": "The minimum battery level (State of Charge/SoC) allowed when arriving at any charging station during your journey.",
-  "charger_departure_soc_description_detailed": "The maximum battery level (State of Charge/SoC) to charge up to at charging stations.",
-  "view_details": "View details",
-  "amenity_add_plan": "Add to plan",
-  "amenity_options": "Amenity options",
-  "add_amentity": "Add amenity",
-  "amenities_wp_button": "Amenity charger around location",
-  "amenities_selection_duration_cost": "{{cost}} to your trip",
-  "amenities_more": "+ {{count}} more {{category}} nearby",
-  "view_charger": "View charger",
-  "amenity_edit_search": "Edit amenity search",
-  "amenity_search_current_of_total": "Amenity serach {{current}} of {{total}}",
-  "amenity_stop": "{{category}} stop",
-  "general_error_message": "An error occurred, please try again."
-}
\ No newline at end of file
+  "card_subscription_fee_per_type": "Subskription: {{fee}} pro {{type}}",
+  "reorder_vehicles": "Fahrzeuge neu sortieren",
+  "edit_order": "Reihenfolge bearbeiten",
+  "amenity_form_timing": "Halt machen",
+  "delete_vehicle_reset_account": "Konto zurücksetzen",
+  "delete_vehicle_reset_account_description": "Dadurch werden alle deine Daten (einschließlich der Fahrzeuge) endgültig gelöscht. Bist du sicher?",
+  "delete_non_tlm_vehicles": "Nicht-TLM-Fahrzeuge löschen",
+  "send": "Senden",
+  "navigation_go_home": "Nach Hause fahren",
+  "departure_soc_description_detailed": "Der Ladezustand der Batterie (State of Charge/SoC) deines Fahrzeugs bei Fahrtantritt.",
+  "destination_arrival_soc_description_detailed": "Der minimale Batteriestand (State of Charge/SoC), der bei der Ankunft an deinem endgültigen Zielort zulässig ist.",
+  "charger_arrival_soc_description_detailed": "Der minimale Batteriestand (State of Charge/SoC), der bei der Ankunft an einer Ladestation während deiner Reise zulässig ist.",
+  "charger_departure_soc_description_detailed": "Der maximale Ladezustand (State of Charge/SoC), bis zu dem an Ladestationen geladen werden soll.",
```

```suggestion
  "charger_departure_soc_description_detailed": "Der maximale Batteriestand (State of Charge/SoC), bis zu dem an Ladestationen geladen werden soll.",
```

Inkonsistent im vergleich zu den beiden darüber. Was ist besser? Ich denke Ladezustand ist einfacher zu verstehen, oder?

---

Ich würde eher bei allen drei zu "Ladestand" tendieren.
Ich sehe es als Pegel(-stand), der beim Wasser die (Füll-)Höhe angibt. ;-)